test/helper.js

```diff
@@ -36,17 +36,17 @@ helper.jsonHttpRequest = function(jsonFile, options) {
     path: '/v1/github',
     method: 'POST',
   };
-
   options = _.defaultsDeep(options, defaultOptions);
-
   let jsonData = JSON.parse(fs.readFileSync(jsonFile));
   options.headers = jsonData.headers;
   return new Promise (function(accept, reject) {
     try {
       let req = http.request(options, accept);
       req.write(JSON.stringify(jsonData.body));
       req.end();
-    } catch (e) {
+      console.log('exiting helper');
+      console.log(accept, reject);
```

Comment:
This `console.log` is executed but the control is still isn't returned to `statusTest` in tests/webhook_test.js` from where it is called.